lib/kitchen/directions/bake_notes/bake_titled_note.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    module BakeTitledNote
+      def self.v1(note:)
+        note.wrap_children(class: 'os-note-body')
+
+        title = note.title&.cut
+
+        note.prepend(child:
+          <<~HTML
+            <h3 class="os-title" data-type="title">
+              <span class="os-title-label">#{note.autogenerated_title}</span>
+            </h3>
+          HTML
+        )
+
+        return unless title
+
+        title.name = 'h4'
+        title.add_class('os-subtitle')
+        title.replace_children(with:
+          <<~HTML
+            <span class="os-subtitle-label">#{title.children}</span>
+          HTML
+        )
+        note.first!('.os-note-body').prepend(child: title.raw)
```

Comment:
I saw someone else do this with `title.raw` ... is the `.raw` required?

Reply:
`prepend` seems to want a `Node`, `NodeSet`, or `String`, and reject a `Kitchen::Element`. `to_s` works just as well as `raw`